docs/v4/TODO.md

```diff
@@ -5,5 +5,8 @@ List of tasks that are important to do before the release of version 4 (but can'
 - [ ] Make migration guide for v3 to v4
 - [ ] Just prior to release: Update conda feedstock recipe dependencies (remove cgen and compiler dependencies). Make sure that recipe is up-to-date.
 - [ ] Revamp the oceanparcels.org landing page, and perhaps also consider new logo/branding?
-- [ ] Look into xarray and whether users can create periodic datasets without increasing the size of the original dataset (i.e., no compromise alternative to `time_periodic` param in v3). Update docs accordingly.
 - [ ] Rerun all the tutorials so that their output is in line with new v4 print statements etc
+- [ ] Remove `tutorial_timestamps.ipynb` tutorial (GH1918)
```

Comment:
Why not do this now already?

Reply:
Though just typing that out that can be done by looking at the v3 docs..... Ok, let's remove :)